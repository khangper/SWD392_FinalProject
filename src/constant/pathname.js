export const PATH_NAME = {
  HOME: "/home",
  EXPLORE: "/explore",
  LIVE_OUTPUT: "/live_output/:id",
  ADD_LIVE_STREAM: "/add_livestream",
  ALL_LIVE_STREAM: "/all_livestream",
  ALL_INSTRUCTOR_LIST: "/allinstructorlist/:id",
  HELP: "/help",
  REPORT_HISTORY: "/reporthistory",
  SEND_FEEDBACK: "/sendfeedback",
  OTHER_INSTRUCTOR_VIEW: "/other-instructor-view/:id",
  MY_INSTRUCTOR_PROFILE: "/my-instructor-profile",
  SAVE_COURSES: "/save-courses",
  SETTING: "/setting",
  MY_COURSE: "/courses",
  LOGIN: "/login",
  FORGOT_PASSWORD: "/forgot-password",
  SIGN_UP: "/sign-up",
  SIGN_UP_STEP: "/SignUpStep",
  THANKS: "/thanks",
  ERROR_404: "/error404",
  INVOICE: "/invoice",
  PRESS: "/press",
  SEARCH_RESULT: "/search_result",
  PAID_MEMBERSHIP: "/paidmembership",
  ABOUT_US: "/about-us",
  OUR_BLOG: "/our-blog",
  BLOG_SINGLE_VIEW: "/blog-single-view",
  CHECKOUT: "/checkout",
  SHOPPING_CART: "/shoppingcart",
  HELP_VIEW_1: "/helpview1",
  HELP_VIEW_2: "/helpview2",
  COMPANY_DETAILS: "/company-details",
  CAREER: "/career",
  APPLY_JOB: "/apply-job",
  CONTACT_US: "/contactus",
  TERMS_OF_USE: "/termsofuse",
  CERTIFICATION_FILL_FORM: "/certificationfillform",
  CERTIFICATION_TEST: "/certificates-test",
  TEST_RESULT: "/test-result",
  MESSAGE: "/messages",
  COURSES_DETAIL_VIEW: "/courses-detail-view/:id",
  PREVIEWS: "/reviews",
  COMING_SOON: "/coming-soon",
  INS_DASHBOARD: "/ins-dashboard",
  PURCHASE_COURSE: "/PurchaseCourse",
  NOTIFICATON: "/notifications",
  DOWLOADCOURSE: "/dowloadcourse",
  CERTIFICATION_CENTER: "/certificationcenter",
  SITEMAP: "/sitemap",
  EARNING: "/earning",
  PAYOUT: "/payout",
  VERTIFICATION: "/vertification",
  MYCERTIFICATIONS: "/mycertifications",
  CREDIT: "/credits",
  STATEMENTIS: "/statementis",
  STATEMENTST: "/statementst",
  ANALYTICS: "/analytics",
  CREATE_NEW_COURSE: "/create-course",
  STD_DASHBOARD: "/student-dashboard",
  NOTIFICATIONSTUDENT: "/notificationsstudent",
};
