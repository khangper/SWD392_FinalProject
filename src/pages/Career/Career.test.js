import React from "react";
import { render, screen } from "@testing-library/react";
import Career from "../Career/Career";
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";

describe("Career", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });
  it("renders the Career component", async () => {
    const { container } = renderWithProviders(<Career />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});
