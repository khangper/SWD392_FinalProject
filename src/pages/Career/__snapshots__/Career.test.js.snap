// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Career renders the Career component 1`] = `
<div>
  <div
    class="career"
  >
    <header
      class="header"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <div
        class="profile"
      >
        <img
          alt=""
          class="header-avatar"
          src="profile-img.jpg"
        />
      </div>
    </header>
    <main>
      <div
        class="career-parent"
      >
        <div
          class="career-background"
        >
          <div
            class="career-navigation-links-parent"
          >
            <div
              class="career-navigation-links"
            >
              <a
                class="career-about"
                href="/about-us"
              >
                About
              </a>
            </div>
            <div
              class="career-navigation-links-1"
            >
              <a
                class="career-blog"
                href="/our-blog"
              >
                Blog
              </a>
            </div>
            <div
              class="career-navigation-links-2"
            >
              <a
                class="career-company"
                href="/company-details"
              >
                Company
              </a>
            </div>
            <div
              class="career-navigation-links-3"
            >
              <a
                class="career-careers"
                href="/career"
              >
                Careers
              </a>
            </div>
            <div
              class="career-navigation-links-4"
            >
              <a
                class="career-press"
                href="/press"
              >
                Press
              </a>
            </div>
          </div>
          <div
            class="career-heading-2-wrapper"
          >
            <h2
              class="career-heading-2"
            >
              We're a passionate, creative, and global company, come work with us.
            </h2>
          </div>
        </div>
        <div
          class="career-working"
        >
          <iframe
            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
            allowfullscreen=""
            class="career-iframe"
            frameborder="0"
            referrerpolicy="strict-origin-when-cross-origin"
            src="https://www.youtube.com/embed/TKnufs85hXk"
            title="Envato Elements: Unlimited Assets For Your Creative Projects"
          />
          <div
            class="career-story-content"
          >
            <div
              class="career-story-wrapper"
            >
              <div
                class="career-heading-2-our-story"
              >
                <h2
                  class="career-heading-23"
                >
                  Working at Cursus
                </h2>
                <div
                  class="career-line-story"
                />
              </div>
              <div
                class="career-our-story-container"
              >
                <p
                  class="career-our-story-content"
                >
                  Phasellus ex mauris, rhoncus quis posuere sit amet, ultricies nec lorem. Vivamus vestibulum porta urna, in placerat lectus facilisis sit amet. Vestibulum non mauris augue. Maecenas arcu magna, aliquam imperdiet tempor nec, lobortis ac erat. Aliquam vel magna tortor. Cras ornare, enim eu tristique tristique, orci nisl blandit mi, at dignissim velit leo nec metus.
                </p>
              </div>
              <button
                class="career-check-out-button"
              >
                Check Out Our Open Roles
              </button>
            </div>
          </div>
        </div>
        <div
          class="career-culture-container-wrapper"
        >
          <div
            class="career-culture-container"
          >
            <button
              class="career-button left"
            />
            <div
              class="career-culture-parent"
            >
              <div
                class="career-image-description"
              >
                <div
                  class="career-link"
                >
                  <img
                    alt=""
                    class="career-image-career"
                    src="Learners.png"
                  />
                  <h4
                    class="career-heading-43"
                  >
                    We are learners
                  </h4>
                  <p
                    class="career-contents"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan. Nunc at tortor tellus. Cras dignissim velit velit, ac sollicitudin mi bibendum in. In vel nibh sodales, venenatis eros a, vulputate ligula.
                  </p>
                </div>
              </div>
              <div
                class="career-image-description"
              >
                <div
                  class="career-link"
                >
                  <img
                    alt=""
                    class="career-image-career"
                    src="Learners.png"
                  />
                  <h4
                    class="career-heading-43"
                  >
                    We are navigators
                  </h4>
                  <p
                    class="career-contents"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan. Nunc at tortor tellus. Cras dignissim velit velit, ac sollicitudin mi bibendum in. In vel nibh sodales, venenatis eros a, vulputate ligula.
                  </p>
                </div>
              </div>
              <div
                class="career-image-description"
              >
                <div
                  class="career-link"
                >
                  <img
                    alt=""
                    class="career-image-career"
                    src="Learners.png"
                  />
                  <h4
                    class="career-heading-43"
                  >
                    We are global
                  </h4>
                  <p
                    class="career-contents"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan. Nunc at tortor tellus. Cras dignissim velit velit, ac sollicitudin mi bibendum in. In vel nibh sodales, venenatis eros a, vulputate ligula.
                  </p>
                </div>
              </div>
              <div
                class="career-image-description"
              >
                <div
                  class="career-link"
                >
                  <img
                    alt=""
                    class="career-image-career"
                    src="Learners.png"
                  />
                  <h4
                    class="career-heading-43"
                  >
                    We make an impact
                  </h4>
                  <p
                    class="career-contents"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan. Nunc at tortor tellus. Cras dignissim velit velit, ac sollicitudin mi bibendum in. In vel nibh sodales, venenatis eros a, vulputate ligula.
                  </p>
                </div>
              </div>
              <div
                class="career-image-description"
              >
                <div
                  class="career-link"
                >
                  <img
                    alt=""
                    class="career-image-career"
                    src="Learners.png"
                  />
                  <h4
                    class="career-heading-43"
                  >
                    We are learners
                  </h4>
                  <p
                    class="career-contents"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan. Nunc at tortor tellus. Cras dignissim velit velit, ac sollicitudin mi bibendum in. In vel nibh sodales, venenatis eros a, vulputate ligula.
                  </p>
                </div>
              </div>
            </div>
            <button
              class="career-button right"
            />
          </div>
        </div>
        <div
          class="career-features-content-wrapper"
        >
          <div
            class="career-features-content"
          >
            <div
              class="career-features-wrapper"
            >
              <div
                class="career-features-inner-wrapper"
              >
                <div
                  class="career-heading-2-our-features"
                >
                  <h3
                    class="career-heading-22"
                  >
                    Benefits
                  </h3>
                </div>
                <div
                  class="career-on-cursus-parent"
                >
                  <div
                    class="career-on-cursus"
                  >
                    Cursus culture is something special, and to complement and support that culture we have some amazing benefits.
                  </div>
                  <div
                    class="career-line-wrapper"
                  />
                </div>
              </div>
            </div>
            <div
              class="career-feature-list"
            >
              <div
                class="career-feature-items"
              >
                <div
                  class="career-icon-parent"
                >
                  <svg
                    class="career-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 256 256"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M200,220H160.73c5.18-5,10.75-10.71,16.33-17.13C205.15,170.57,220,136.37,220,104a92,92,0,0,0-184,0c0,50,34.12,91.94,59.18,116H56a12,12,0,0,0,0,24H200a12,12,0,0,0,0-24ZM60,104a68,68,0,0,1,136,0c0,33.31-20,63.37-36.7,82.71A249.35,249.35,0,0,1,128,216.89a249.35,249.35,0,0,1-31.3-30.18C80,167.37,60,137.31,60,104Zm68,44a44,44,0,1,0-44-44A44.05,44.05,0,0,0,128,148Zm0-64a20,20,0,1,1-20,20A20,20,0,0,1,128,84Z"
                    />
                  </svg>
                </div>
                <div
                  class="career-feature-descriptions"
                >
                  <div
                    class="career-heading-4-wrapper"
                  >
                    <div
                      class="career-heading-4"
                    >
                      Work from anywhere
                    </div>
                  </div>
                  <div
                    class="career-container"
                  >
                    <p
                      class="career-content"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="career-feature-items"
              >
                <div
                  class="career-icon-parent"
                >
                  <svg
                    class="career-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 256 256"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M185.33,114.21l29.14-27.42.17-.17a32,32,0,0,0-45.26-45.26c0,.06-.11.11-.17.17L141.79,70.67l-83-30.2a8,8,0,0,0-8.39,1.86l-24,24a8,8,0,0,0,1.22,12.31l63.89,42.59L76.69,136H56a8,8,0,0,0-5.65,2.34l-24,24A8,8,0,0,0,29,175.42l36.82,14.73,14.7,36.75.06.16a8,8,0,0,0,13.18,2.47l23.87-23.88A8,8,0,0,0,120,200V179.31l14.76-14.76,42.59,63.89a8,8,0,0,0,12.31,1.22l24-24a8,8,0,0,0,1.86-8.39Zm-.07,97.23-42.59-63.88A8,8,0,0,0,136.8,144c-.27,0-.53,0-.79,0a8,8,0,0,0-5.66,2.35l-24,24A8,8,0,0,0,104,176v20.69L90.93,209.76,79.43,181A8,8,0,0,0,75,176.57l-28.74-11.5L59.32,152H80a8,8,0,0,0,5.66-2.34l24-24a8,8,0,0,0-1.22-12.32L44.56,70.74l13.5-13.49,83.22,30.26a8,8,0,0,0,8.56-2L180.78,52.6A16,16,0,0,1,203.4,75.23l-32.87,30.93a8,8,0,0,0-2,8.56l30.26,83.22Z"
                    />
                  </svg>
                </div>
                <div
                  class="career-feature-descriptions"
                >
                  <div
                    class="career-heading-4-wrapper"
                  >
                    <div
                      class="career-heading-4"
                    >
                      Work and Travel
                    </div>
                  </div>
                  <div
                    class="career-container"
                  >
                    <p
                      class="career-content"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="career-feature-items"
              >
                <div
                  class="career-icon-parent"
                >
                  <svg
                    class="career-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 256 256"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M128,24A104,104,0,1,0,232,128,104.11,104.11,0,0,0,128,24Zm8,16.37a86.4,86.4,0,0,1,16,3V212.67a86.4,86.4,0,0,1-16,3Zm32,9.26a87.81,87.81,0,0,1,16,10.54V195.83a87.81,87.81,0,0,1-16,10.54ZM40,128a88.11,88.11,0,0,1,80-87.63V215.63A88.11,88.11,0,0,1,40,128Zm160,50.54V77.46a87.82,87.82,0,0,1,0,101.08Z"
                    />
                  </svg>
                </div>
                <div
                  class="career-feature-descriptions"
                >
                  <div
                    class="career-heading-4-wrapper"
                  >
                    <div
                      class="career-heading-4"
                    >
                      Flexible Hours
                    </div>
                  </div>
                  <div
                    class="career-container"
                  >
                    <p
                      class="career-content"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="career-feature-items"
              >
                <div
                  class="career-icon-parent"
                >
                  <svg
                    class="career-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 496 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M248 8C111 8 0 119 0 256s111 248 248 248 248-111 248-248S385 8 248 8zm0 448c-110.3 0-200-89.7-200-200S137.7 56 248 56s200 89.7 200 200-89.7 200-200 200zm-80-216c17.7 0 32-14.3 32-32s-14.3-32-32-32-32 14.3-32 32 14.3 32 32 32zm160 0c17.7 0 32-14.3 32-32s-14.3-32-32-32-32 14.3-32 32 14.3 32 32 32zm4 72.6c-20.8 25-51.5 39.4-84 39.4s-63.2-14.3-84-39.4c-8.5-10.2-23.7-11.5-33.8-3.1-10.2 8.5-11.5 23.6-3.1 33.8 30 36 74.1 56.6 120.9 56.6s90.9-20.6 120.9-56.6c8.5-10.2 7.1-25.3-3.1-33.8-10.1-8.4-25.3-7.1-33.8 3.1z"
                    />
                  </svg>
                </div>
                <div
                  class="career-feature-descriptions"
                >
                  <div
                    class="career-heading-4-wrapper"
                  >
                    <div
                      class="career-heading-4"
                    >
                      Purchasing Leave
                    </div>
                  </div>
                  <div
                    class="career-container"
                  >
                    <p
                      class="career-content"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="career-feature-items"
              >
                <div
                  class="career-icon-parent"
                >
                  <svg
                    class="career-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0V0zm0 0h24v24H0V0z"
                      fill="none"
                    />
                    <path
                      d="M1 18v3h3c0-1.66-1.34-3-3-3zm0-4v2c2.76 0 5 2.24 5 5h2c0-3.87-3.13-7-7-7zm0-4v2a9 9 0 0 1 9 9h2c0-6.08-4.93-11-11-11zm20-7H3c-1.1 0-2 .9-2 2v3h2V5h18v14h-7v2h7c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM5 7v2h12v6h-3v2h5V7z"
                    />
                  </svg>
                </div>
                <div
                  class="career-feature-descriptions"
                >
                  <div
                    class="career-heading-4-wrapper"
                  >
                    <div
                      class="career-heading-4"
                    >
                      Social Events
                    </div>
                  </div>
                  <div
                    class="career-container"
                  >
                    <p
                      class="career-content"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="career-feature-items"
              >
                <div
                  class="career-icon-parent"
                >
                  <svg
                    class="career-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 256 256"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M210.35,129.36c-.81-.47-1.7-.92-2.62-1.36.92-.44,1.81-.89,2.62-1.36a40,40,0,1,0-40-69.28c-.81.47-1.65,1-2.48,1.59.08-1,.13-2,.13-3a40,40,0,0,0-80,0c0,.94,0,1.94.13,3-.83-.57-1.67-1.12-2.48-1.59a40,40,0,1,0-40,69.28c.81.47,1.7.92,2.62,1.36-.92.44-1.81.89-2.62,1.36a40,40,0,1,0,40,69.28c.81-.47,1.65-1,2.48-1.59-.08,1-.13,2-.13,2.95a40,40,0,0,0,80,0c0-.94-.05-1.94-.13-2.95.83.57,1.67,1.12,2.48,1.59A39.79,39.79,0,0,0,190.29,204a40.43,40.43,0,0,0,10.42-1.38,40,40,0,0,0,9.64-73.28ZM104,128a24,24,0,1,1,24,24A24,24,0,0,1,104,128Zm74.35-56.79a24,24,0,1,1,24,41.57c-6.27,3.63-18.61,6.13-35.16,7.19A40,40,0,0,0,154.53,98.1C163.73,84.28,172.08,74.84,178.35,71.21ZM128,32a24,24,0,0,1,24,24c0,7.24-4,19.19-11.36,34.06a39.81,39.81,0,0,0-25.28,0C108,75.19,104,63.24,104,56A24,24,0,0,1,128,32ZM44.86,80a24,24,0,0,1,32.79-8.79c6.27,3.63,14.62,13.07,23.82,26.89A40,40,0,0,0,88.81,120c-16.55-1.06-28.89-3.56-35.16-7.18A24,24,0,0,1,44.86,80ZM77.65,184.79a24,24,0,1,1-24-41.57c6.27-3.63,18.61-6.13,35.16-7.19a40,40,0,0,0,12.66,21.87C92.27,171.72,83.92,181.16,77.65,184.79ZM128,224a24,24,0,0,1-24-24c0-7.24,4-19.19,11.36-34.06a39.81,39.81,0,0,0,25.28,0C148,180.81,152,192.76,152,200A24,24,0,0,1,128,224Zm83.14-48a24,24,0,0,1-32.79,8.79c-6.27-3.63-14.62-13.07-23.82-26.89A40,40,0,0,0,167.19,136c16.55,1.06,28.89,3.56,35.16,7.18A24,24,0,0,1,211.14,176Z"
                    />
                  </svg>
                </div>
                <div
                  class="career-feature-descriptions"
                >
                  <div
                    class="career-heading-4-wrapper"
                  >
                    <div
                      class="career-heading-4"
                    >
                      Wellness Program
                    </div>
                  </div>
                  <div
                    class="career-container"
                  >
                    <p
                      class="career-content"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="career-feature-items"
              >
                <div
                  class="career-icon-parent"
                >
                  <svg
                    class="career-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 256 256"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M252.51,108.8,227,57.75a14,14,0,0,0-18.78-6.27L182.66,64.26,129.53,50.2a6.1,6.1,0,0,0-3.06,0L73.34,64.26,47.79,51.48A14,14,0,0,0,29,57.75L3.49,108.8a14,14,0,0,0,6.26,18.78L36.9,141.16l55.61,39.72a6,6,0,0,0,2,.94l64,16A6.08,6.08,0,0,0,160,198a6,6,0,0,0,4.24-1.76l55.31-55.31,26.7-13.35a14,14,0,0,0,6.26-18.78Zm-53,35.16-35.8-28.68a6,6,0,0,0-8,.45c-18.65,18.79-39.5,16.42-52.79,7.92a2,2,0,0,1-.94-1.5,1.9,1.9,0,0,1,.51-1.55L146.43,78h33.86l28.41,56.82ZM14.11,115.69a2,2,0,0,1,.11-1.52L39.74,63.11a2,2,0,0,1,1.8-1.1,2,2,0,0,1,.89.21l22.21,11.1L37.32,128l-22.21-11.1A2,2,0,0,1,14.11,115.69Zm144.05,69.67-59.6-14.9L47.66,134.1,76.84,75.75,128,62.21l14.8,3.92a5.92,5.92,0,0,0-3,1.57L94.1,112.05a14,14,0,0,0,2.39,21.72c20.22,12.92,44.75,10.49,63.8-5.89L191,152.5Zm83.73-69.67a2,2,0,0,1-1,1.16L218.68,128,191.36,73.32l22.21-11.1a2,2,0,0,1,1.53-.11,2,2,0,0,1,1.16,1l25.52,51.06A2,2,0,0,1,241.89,115.69Zm-112,101.76a6,6,0,0,1-7.27,4.37L80.89,211.39a5.88,5.88,0,0,1-2-.94L52.52,191.64a6,6,0,1,1,7-9.77L84.91,200l40.61,10.15A6,6,0,0,1,129.88,217.45Z"
                    />
                  </svg>
                </div>
                <div
                  class="career-feature-descriptions"
                >
                  <div
                    class="career-heading-4-wrapper"
                  >
                    <div
                      class="career-heading-4"
                    >
                      Mentoring Program
                    </div>
                  </div>
                  <div
                    class="career-container"
                  >
                    <p
                      class="career-content"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="career-feature-items"
              >
                <div
                  class="career-icon-parent"
                >
                  <svg
                    class="career-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    version="1.1"
                    viewBox="0 0 17 17"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <g />
                    <path
                      d="M15.291 12.592l-1.57-3.127-3.127 1.568 0.448 0.895 1.871-0.939c-0.373 1.907-1.956 3.365-3.913 3.585v-7.574h1.5v-1h-1.5v-1.051c1.14-0.232 2-1.241 2-2.449 0-1.379-1.122-2.5-2.5-2.5s-2.5 1.121-2.5 2.5c0 1.208 0.86 2.217 2 2.449v1.051h-1.5v1h1.5v7.574c-1.955-0.22-3.538-1.676-3.912-3.58l1.86 0.934 0.448-0.895-3.127-1.568-1.57 3.127 0.894 0.449 0.651-1.298c0.693 2.245 2.787 3.882 5.256 3.882 2.464 0 4.556-1.63 5.252-3.867l0.645 1.283 0.894-0.449zM7 2.5c0-0.827 0.673-1.5 1.5-1.5s1.5 0.673 1.5 1.5-0.673 1.5-1.5 1.5-1.5-0.673-1.5-1.5z"
                    />
                  </svg>
                </div>
                <div
                  class="career-feature-descriptions"
                >
                  <div
                    class="career-heading-4-wrapper"
                  >
                    <div
                      class="career-heading-4"
                    >
                      Fundraising
                    </div>
                  </div>
                  <div
                    class="career-container"
                  >
                    <p
                      class="career-content"
                    >
                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eget enim in turpis consequat tempor sed id neque. Nam at felis et elit auctor accumsan.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="career-open-roles-wrapper"
        >
          <div
            class="career-open-roles-content"
          >
            <div
              class="career-features-wrapper"
            >
              <div
                class="career-features-inner-wrapper"
              >
                <div
                  class="career-heading-2-our-features"
                >
                  <h3
                    class="career-heading-22"
                  >
                    Open Roles
                  </h3>
                </div>
                <div
                  class="career-on-cursus-parent"
                >
                  <div
                    class="career-on-cursus"
                  >
                    Cursus is a fast growing company and we're expanding both our Punjab office and international offices.
                  </div>
                  <div
                    class="career-line-wrapper"
                  />
                </div>
              </div>
            </div>
            <div
              class="career-job-listings"
            >
              <div
                class="career-job-listing"
              >
                <div
                  class="career-listing-details"
                >
                  <div
                    class="career-apply-parent"
                  >
                    <div
                      class="career-listing-title-container"
                    >
                      <div
                        class="career-heading-45"
                      >
                        Algorithm Engineer
                      </div>
                    </div>
                    <div
                      class="career-listing-location-container"
                    >
                      <div
                        class="career-country-open"
                      >
                        India
                      </div>
                    </div>
                    <a
                      href="/apply-job"
                    >
                      <button
                        class="career-button-learn-more"
                      >
                        Learn More and Apply
                      </button>
                    </a>
                  </div>
                  <div
                    class="career-apply-parent"
                  >
                    <div
                      class="career-listing-title-container"
                    >
                      <div
                        class="career-heading-45"
                      >
                        Chief Technology Officer
                      </div>
                    </div>
                    <div
                      class="career-listing-location-container"
                    >
                      <div
                        class="career-country-open"
                      >
                        India
                      </div>
                    </div>
                    <a
                      href="/apply-job"
                    >
                      <button
                        class="career-button-learn-more"
                      >
                        Learn More and Apply
                      </button>
                    </a>
                  </div>
                  <div
                    class="career-apply-parent"
                  >
                    <div
                      class="career-listing-title-container"
                    >
                      <div
                        class="career-heading-45"
                      >
                        Algorithm Engineer
                      </div>
                    </div>
                    <div
                      class="career-listing-location-container"
                    >
                      <div
                        class="career-country-open"
                      >
                        India
                      </div>
                    </div>
                    <a
                      href="/apply-job"
                    >
                      <button
                        class="career-button-learn-more"
                      >
                        Learn More and Apply
                      </button>
                    </a>
                  </div>
                  <div
                    class="career-apply-parent"
                  >
                    <div
                      class="career-listing-title-container"
                    >
                      <div
                        class="career-heading-45"
                      >
                        Algorithm Engineer
                      </div>
                    </div>
                    <div
                      class="career-listing-location-container"
                    >
                      <div
                        class="career-country-open"
                      >
                        India
                      </div>
                    </div>
                    <a
                      href="/apply-job"
                    >
                      <button
                        class="career-button-learn-more"
                      >
                        Learn More and Apply
                      </button>
                    </a>
                  </div>
                  <div
                    class="career-apply-parent"
                  >
                    <div
                      class="career-listing-title-container"
                    >
                      <div
                        class="career-heading-45"
                      >
                        Algorithm Engineer
                      </div>
                    </div>
                    <div
                      class="career-listing-location-container"
                    >
                      <div
                        class="career-country-open"
                      >
                        San Francisco, CA
                      </div>
                    </div>
                    <a
                      href="/apply-job"
                    >
                      <button
                        class="career-button-learn-more"
                      >
                        Learn More and Apply
                      </button>
                    </a>
                  </div>
                  <div
                    class="career-apply-parent"
                  >
                    <div
                      class="career-listing-title-container"
                    >
                      <div
                        class="career-heading-45"
                      >
                        Algorithm Engineer
                      </div>
                    </div>
                    <div
                      class="career-listing-location-container"
                    >
                      <div
                        class="career-country-open"
                      >
                        India
                      </div>
                    </div>
                    <a
                      href="/apply-job"
                    >
                      <button
                        class="career-button-learn-more"
                      >
                        Learn More and Apply
                      </button>
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="footer footer-large-container"
    >
      <div
        class="footer-links"
      >
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/about-us"
          >
            About
          </a>
          <a
            class="link"
            href="/blog-single-view/:id"
          >
            Blog
          </a>
          <a
            class="link"
            href="/career"
          >
            Careers
          </a>
          <a
            class="link"
            href="/press"
          >
            Press
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/help"
          >
            Help
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Advertise
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Developers
          </a>
          <a
            class="link"
            href="/contactus"
          >
            Contact Us
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/termsofuse"
          >
            Copyright Policy
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Terms
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Privacy Policy
          </a>
          <a
            class="link"
            href="/sitemap"
          >
            Sitemap
          </a>
        </div>
        <div
          class="footer-column"
        >
          <button
            class="teach-button"
          >
            Teach on Cursus
          </button>
          <div
            class="language-selector"
          >
            <button>
              Language 
              <span>
                ▼
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer-bottom"
      >
        <div
          class="footer-copy"
        >
          <img
            alt=""
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
          />
          © 2024 
          <strong>
            Cursus
          </strong>
          . All Rights Reserved.
        </div>
        <div
          class="footer-social"
        >
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="facebook.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="x.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="google+.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="linkedin.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="insta.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="pinterest.png"
            />
          </a>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
